simple_settings/core.py

```diff
@@ -71,8 +71,14 @@ def setup(self):
     def _load_settings_pipeline(self):
         for settings_file in self._settings_list:
             strategy = self._get_strategy_by_file(settings_file)
-            settings = strategy.load_settings_file(settings_file)
-            self._dict.update(settings)
+            try:
+                settings = strategy.load_settings_file(settings_file)
+            except Exception:
+                print('Error processing settings_file: {}'.format(
```

Comment:
what about:
```python
import logging
...
logger = logging.getLogger(__name__)
...
except Exception as e:
    logger.error(
        'Error processing settings_file "{}": {}'.format(
             settings_file, e
         )
    )
```
?
By default, the logger will log on stderr but the user can customize with logging configuration.